src/platforms/hosted/ftdi_bmp.c

```diff
@@ -592,7 +612,7 @@ size_t libftdi_buffer_write(const uint8_t *data, size_t size)
 	return size;
```

Comment:
sorry can't add comments to unedited lines
missing `U`:
lines 517 throughh 536
line 575